magma/inline_verilog.py

```diff
@@ -58,8 +50,10 @@ class InlineVerilogError(RuntimeError):
 
 def _insert_temporary_wires(cls, value, inline_wire_prefix):
     """
-    For non DefnRef, insert a temporary Wire instance so the signal isn't
-    inlined out
+    Insert a temporary Wire instance so the signal isn't inlined out
+
+    We have to do this for DefnRef because the coreir inline.cpp logic
+    sometimes inserts temporary wires for DefnRef that eventually get inlined
```

Comment:
nit
```suggestion
    Insert a temporary Wire instance so the signal isn't inlined out.

    We have to do this for DefnRef because the coreir inline.cpp logic
    sometimes inserts temporary wires for DefnRef that eventually get inlined.
```